strr-web/pages/create-account.vue

```diff
@@ -206,7 +206,7 @@ const validateStep = (schema: any, state: any, index: number) => {
 }
 
 const validatePropertyManagerStep = () => {
-  if (!formState.isPropertyManagerRole && !formState.hasPropertyManager) {
+  if (!formState.isPropertyManagerRole) {
```

Comment:
Stepper validation is breaking with this formData
![image](https://github.com/user-attachments/assets/e7ff8c64-5983-401a-a9d6-f90992d322ee)

Reply:
Fixed